src/github.com/getlantern/flashlight/server/server.go

```diff
@@ -240,7 +241,7 @@ func determineInternalIP() (string, error) {
 }
 
 func onBytesGiven(destAddr string, req *http.Request, bytes int64) {
-	_, port, _ := net.SplitHostPort(destAddr)
+	host, port, _ := net.SplitHostPort(destAddr)
```

Comment:
We actually don't care whether this succeeds or not.  If it fails, we get a blank host and 0 port, both of which we record anyway.